tests/system/small/test_dataframe.py

```diff
@@ -1020,13 +1020,20 @@ def test_df_interpolate(scalars_dfs):
     )
 
 
-def test_df_fillna(scalars_dfs):
+@pytest.mark.parametrize(
```

Comment:
Let's add the customer usecase of filling "None" as a test

Reply:
That should be the same as filling "A" I assume?

Reply:
fair enough!